header.py

```diff
@@ -7,4 +9,22 @@ def __init__(self, content, level = 1):
         self._level = level
     
     def render(self):
-        return "<h{0}> {1} </h{0}>".format(self._level, self.content())
+        return "<h{0}>{1}</h{0}>".format(self._level, self.content())
+
+
+
+def parser(content: typing.Text) -> (int, int, Header):
```

Comment:
what about make this function as `Header`'s static method? Otherwise when we build `Parser` class, there will be so many `parser()` to be imported and needed to set up alias by using `as`.

Reply:
SG, maybe fix in following pr.

Reply:
it seems like python doesn't comfortable with a class referring itself.